spring-remoting/remoting-http/server/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When I imported the projects into Eclipse, I am getting the following error related to this pom.xml file:
org.codehaus.plexus.archiver.jar.Manifest.write(java.io.PrintWriter)

Are you seeing the same thing?